docs/index.rst

```diff
@@ -1,159 +1,325 @@
 Testimony
 =========
 
-An approach to document test cases in the Python source code
-------------------------------------------------------------
+.. contents:: Topics
 
-Are you tired of managing your test cases in a test case management tool and
-your test code in a Python automation framework?  Testimony can help you to use
-your Python automation framework as a test case repository tool.
+.. _what_is_testimony:
```

Comment:
Here's what seems to work:

```rst
===============
Title Goes Here
===============

.. contents:: Table of Contents
   :backlinks: entry

Topic 1
=======
Blah blah blah

Subtopic A
----------

Topic 2
=======

Topic 3
=======
```

This generates the following page:

![screenshot 2016-07-12 11 26 46](https://cloud.githubusercontent.com/assets/53362/16772805/8c77b1b0-4823-11e6-8ec6-91d8c48d5721.png)